phys2bids/physio_obj.py

```diff
@@ -162,17 +170,55 @@ def ch_amount(self):
         """
         return len(self.timeseries)
 
-    def rename_channels(self, new_names, ch_trigger=None):
+    def __getitem__(self, idx):
         """
-        Renames the channels. If 'time' or 'trigger' were specified,
-        it makes sure that they're the first and second entry.
+        Return a copy of the object with a sliced version of self.timeseries.
+
+        The slicing is based on the trigger. If necessary, computes a sort of
+        interpolation to get the right index in multifreq.
+
+        Parameters
+        ----------
+        idx: int, tuple of int or slicer
+            indexes to use to slice the timeseries.
+
+        Returns
+        -------
+        BlueprintInput object
+            a copy of the object with the part of timeseries expressed by idx.
+        """
+        sliced_timeseries = []
+
+        if isinstance(idx, int):
+            idx = slice(idx, idx + 1)
+
+        if not self.trigger_idx:
+            self.trigger_idx = 0
+
+        for n, channel in enumerate(self.timeseries):
+            idx_dict = {'start': idx.start, 'stop': idx.stop, 'step': idx.step}
```

Comment:
So, for our purposes in multi-run processing : step will most likely be `0` right ?

Reply:
Indeed! But this little function makes us be able to just have a standard indexing:
```
phys_in[230:360]
```

That's the main idea at least. Now I need to test it before we can actually use it!